docker/run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Run tests.
+#
+# Run this in the app container.
+
+TESTING=true pytest "$@"
```

Comment:
Even if we move ``TESTING=true`` to ``test.env``, I think it is useful to manually set it here.